pkg/simpleschema/markers_test.go

```diff
@@ -124,6 +124,15 @@ func TestParseMarkers(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name:  "validation marker with one value",
```

Comment:
Can you add a test for multiple validation markers, and a test for using quotes (e.g. `self == "foobar"`)?